### PR TITLE
Fix build warnings

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/PageDirectiveTest.cs [hunk 1]
```diff
@@ -28,24 +28,6 @@ public void TryGetPageDirective_ReturnsFalse_IfPageDoesNotHaveDirective()
             Assert.Null(pageDirective);
         }
 
-        [Fact(Skip = "https://github.com/aspnet/Razor/issues/1201")]
-        public void TryGetPageDirective_ReturnsFalse_IfPageDoesStartWithDirective()
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/PageDirectiveTest.cs]: @rynowak do you want to keep this?

Comment (rynowak) [inline test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/PageDirectiveTest.cs]: Does this still fail? This should pass now

Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/PageDirectiveTest.cs]: Yup, it does pass. Unskipped

File: build/dependencies.props [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+﻿<Project>
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
     <BenchmarkDotNetVersion>0.10.3</BenchmarkDotNetVersion>
     <CoreFxVersion>4.4.0-*</CoreFxVersion>
     <DependencyModelVersion>2.0.0-*</DependencyModelVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <JsonNetVersion>10.0.1</JsonNetVersion>
-    <MoqVersion>4.7.1</MoqVersion>
+    <MoqVersion>4.7.49</MoqVersion>
```
Comment (ajaybhargavb) [inline build/dependencies.props]: This is already done https://github.com/aspnet/Razor/pull/1479/files#diff-30dcc9789f8ea3c7632e4f054e60c072R9

**Pull request merged**